state/accountsDB.go

```diff
@@ -1102,56 +1102,58 @@ func (adb *AccountsDB) SnapshotState(rootHash []byte) {
 
 	trieStorageManager.EnterPruningBufferingMode()
 
+	errChan := make(chan error, 1)
```

Comment:
Shouldn't this channel be closed somewhere?

Reply:
not required